src/commands/countdown/subcommands/delete.ts

```diff
@@ -0,0 +1,68 @@
+import dayjs from 'dayjs';
+import * as Discord from '../../../Discord';
+import SubCommand from '../../../SubCommand';
+import CommandOption from '../../../CommandOption';
+import { bold } from '../../../format';
+import {
+  respondWithInvalidDate,
+  respondWithNotFound,
+  validateDate,
+} from '../_common';
+
+const del = new SubCommand({
+  name: 'delete',
+  displayName: 'Delete Countdown',
+  description: 'Delete a countdown',
+  requiredPermissions: [Discord.Permission.MANAGE_ROLES],
+  options: [
+    new CommandOption({
+      name: 'date',
+      description: 'Date of event (MM/DD/YY)',
+      required: true,
+      type: Discord.CommandOptionType.STRING,
+    }),
+    new CommandOption({
+      name: 'name',
+      description: 'Name of the event to delete',
+      required: true,
+      type: Discord.CommandOptionType.STRING,
+    }),
```

Comment:
Can we use some form of ID (located via list command) to more easily delete these, similar to birthday messages?

Reply:
For now, I think we can deploy with this and fast-follow later, since countdowns won't be deleted frequently